### PR TITLE
Adds Sudoku Game

### LINKED ISSUE #296: Add Suduko Game Flutter
https://github.com/einsitang/sudoku-flutter @ashitaprasad

### PR DESCRIPTION
**Pull request by arunimaChintu**
Closes #296 
This adds Sudoku game.

### CODE REVIEW

**Comment by arunimaChintu** [comment]
@ashitaprasad Please merge this.

**Comment by arunimaChintu** [comment]
@ashitaprasad Please merge this.

File: README.md [hunk 1]
```diff
@@ -145,8 +145,10 @@ All the projects added in this project are featured in [fluttergems.dev](flutter
 | Flutter Crush | [Link](https://github.com/boeledi/flutter_crush) | Match 3 game like Candy Crush. |
 | 2048 | [Link](https://github.com/shubhexists/2048) |2048 game. |
 | Kunai Game | [Link](https://github.com/hzeyuan/flutter-flame-kunaiGame) | A simple Kunai game. |
+| Suduko Game | [Link](https://github.com/einsitang/sudoku-flutter) | A sudoku flutter Game |
 | Tetris | [Link](https://github.com/priyanshudutta04/Tetris) | A Tetris game. |
 
+
```
Comment (ashitaprasad) [inline README.md:151]: Any reason for adding an extra new line here ?

**Comment by arunimaChintu** [comment]
@ashitaprasad removed the extra line. Please merge it now

**Comment by ashitaprasad** [comment]
The project already exists in the games section.

**Pull request closed without merging**